system-tests/projects/config-cjs-and-esm/config-with-ts-module/cypress.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'cypress'
+
+const fn = async (_, config) => {
+  const ret = await import('find-up')
+
+  console.log(ret)
```

Comment:
Do we need this console.log?

Reply:
Not really. It was helpful to make sure it wasn't getting stripped.